src/shared/file_op.c

```diff
@@ -474,7 +474,7 @@ int CreatePID(const char *name, int pid)
     }
 
     fprintf(fp, "%d\n", pid);
-
+    fflush(fp);
```

Comment:
Why is this necessary?

Reply:
It wasn't strictly necessary. We wanted to increase the speed of creation of the file containing the PID but it didn't make a difference. I'll remove this line.